indico/htdocs/sass/custom/_jquery-ui.scss

```diff
@@ -19,6 +19,10 @@
     @include border-radius();
 }
 
+.ui-dialog {
+    box-shadow: 0px 0px 24px rgba(0, 0, 0, 0.3);
```

Comment:
From the compass documentation: ["[The box-shadow mixin] provides cross-browser for Webkit, Gecko, and CSS3 box shadows when one or more box shadows are needed. Each shadow argument should adhere to the standard css3 syntax for the box-shadow property."](http://compass-style.org/reference/compass/css3/box_shadow/#mixin-box-shadow).

As far as I know CSS3 box-shadows are [supported without vendor prefixes](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#Browser_compatibility)  since quite some time in the major browsers.

So I don't see any advantage at using that mixin.